crates/bevy_reflect/src/lib.rs

```diff
@@ -26,6 +28,7 @@ mod impls {
 
 pub mod serde;
 pub mod std_traits;
+pub mod utility;
```

Comment:
Did not include this module in the prelude. Is this okay/ I think it makes sense to exclude it since right now it only contains code that is unlikely to be used directly.

Reply:
Yes please.